vnext/package.json

```diff
@@ -21,7 +21,7 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.4.0",
-    "@react-native-community/cli": "^2.6.0",
+    "@react-native-community/cli": "^3.0.4",
```

Comment:
I'm not sure that this version of the cli is compatible with react-native 0.60.  We might have to wait until we update react-native before we start using the v3 version of the cli.  
 (Not totally sure though)

Reply:
In fact the dependencies of the cli package say it works with RN ^0.61, so I suspect we need to wait until we are on react-native 0.61 before we update the cli